### PR TITLE
Add pprof debug endpoints to sinker

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/assign @cjwagner @fejta 
/cc @BenTheElder @krzyzacy

### CODE REVIEW

File: prow/cmd/sinker/main.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ package main
 import (
 	"flag"
 	"fmt"
+	_ "net/http/pprof"
```
Comment (fejta) [inline prow/cmd/sinker/main.go]: doesn't this also need an http server?

Comment (stevekuznetsov) [reply prow/cmd/sinker/main.go]: :man_facepalming:

**Comment by stevekuznetsov** [comment]
@fejta performance critical parts ok k8s run with it on, so I think we should be able to keep it forever, yes

**Comment by BenTheElder** [comment]
[catching up on notification backlog] 👍

**Pull request merged**